pkg/management/postgres/probes.go

```diff
@@ -86,6 +92,30 @@ func (instance *Instance) GetStatus() (result *postgres.PostgresqlStatus, err er
 		return result, err
 	}
 
+	// Get the latest configuration hash
+	rowConfigHash := superUserDB.QueryRow(
+		fmt.Sprintf("SHOW %s", pgx.Identifier{postgres.CNPGConfigSha256}.Sanitize()))
+	if err := rowConfigHash.Scan(&result.LoadedConfigurationHash); err != nil {
+		var pgErr *pgconn.PgError
+		if errors.As(err, &pgErr) && pgErr.Code == SQLStateUndefinedObject {
+			// The applied configuration has not been generated by
+			// CloudNativePG. This could happen if PostgreSQL starts
+			// and we still didn't load a generated configuration.
+			//
+			// This unfortunately will report an error
+			// into the PostgreSQL logs, but there's no
+			// current way to do it without raising an
+			// error at this moment.
+			//
+			// It should not represent a major issue since
+			// a correct configuration is going to be
+			// loaded immediately.
```

Comment:
Here I wonder. So, we set the configuration hash to "", and then when we do `reconcilePods`, we see that not all pods have a hash, so we do a rolling upgrade, and this could repeat?